common/libzkp/e2e-test.sh

```diff
@@ -0,0 +1,44 @@
+set -xeu
+set -o pipefail
+
+export CHAIN_ID=534352
+export RUST_BACKTRACE=full
+export RUST_LOG=debug
+export RUST_MIN_STACK=100000000
+export PROVER_OUTPUT_DIR=test_zkp_test
+#export LD_LIBRARY_PATH=/:/usr/local/cuda/lib64
+
+mkdir -p $PROVER_OUTPUT_DIR
+
+REPO=$(realpath ../..)
+
+function build_test_bins() {
+    cd impl
+    cargo build --release
+    ln -f -s $(realpath target/release/libzkp.so) $REPO/prover/core/lib
+    ln -f -s $(realpath target/release/libzkp.so) $REPO/coordinator/internal/logic/verifier/lib
+    cd $REPO/prover
+    go test -tags="gpu ffi" -timeout 0 -c core/prover_test.go
```

Comment:
is this file used? since core/prover_test.go is invalid now

Reply:
makes sense, emmm, but we have not added the rust version of this e2e-test script. what about removing them in this PR, and add one in next PR?

Reply:
It's not the main part, can be leaved to next patch

Reply:
i think the script is not useful now.. can be deleted later